src/components/AddItem.js

```diff
@@ -1,11 +1,102 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { db } from '../lib/firebase';
+
+const initialFormState = {
+  itemName: '',
+  timeFrame: 7,
+  lastPurchased: null,
+};
 
 const AddItem = () => {
+  const token = 'user token';
+  const history = useHistory();
+  const [formData, setFormData] = useState(initialFormState);
+  const [error, setError] = useState('');
+  console.log(error);
+
```

Comment:
When I test out the form in the browser, I'm not able to change the radio button.  I think that would be controlled with state as well, so you could do something like
`  const [radioValue, setRadioValue] = useState('Soon');`

then add a helper function to update radioValue
```
  const handleRadioChange = event => {
    setRadioValue(event.target.value);
  }
```
and then...

Reply:
Yea, there was an issue with changing the radio buttons, we had to click it twice for the UI to change, but the state is changing (we checked in Components tab of chrome's react dev tools)

Reply:
I think we may have to check the `checked` prop to a value of true or false like they do [here](http://react.tips/radio-buttons-in-reactjs/)